src/sugar3/activity/activity.py

```diff
@@ -19,6 +21,9 @@ class ReadActivity(activity.Activity):
         pass
 
 To get a real, working activity, you will at least have to implement:
+
+.. code-block:: python
+
     __init__(), read_file() and write_file()
```

Comment:
Why do we need to make this a code block?  Could we make these items links and part of the paragraph?  :func:`sugar3.activity.activity.Activity.__init__`